src/style.css

```diff
@@ -0,0 +1,38 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars-star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:nth-child(1),
```

Comment:
you can use a formula like :nth-child(-n + #stars-number) to achieve the same in a cleaner way